### PR TITLE
Support explicit Exit MetaCommand in REPL

### PR DESCRIPTION
**Pull request by Nirmal4G**
Instead of exiting the `msi` by newline, we exit through `#exit` meta command in REPL

### CODE REVIEW

File: src/msi/Repl.cs [hunk 1]
```diff
@@ -432,6 +432,15 @@ private void EvaluateMetaCommand(string input)
             var inQuotes = false;
             var position = 1;
             var sb = new StringBuilder();
+
+            void CommitPendingArgument()
```
Comment (terrajobst) [inline src/msi/Repl.cs:436]: Why did this function move? Seems unrelated.

**Review by terrajobst — CHANGES_REQUESTED** [review]

File: src/msi/Repl.cs (hunk 1 above)
Comment (Nirmal4G) [reply src/msi/Repl.cs:436]: I was checking if the order of declaration matters... I forgot to discard the change after I've checked it. Sorry about that!

Comment (Nirmal4G) [reply src/msi/Repl.cs]: Fixed it!

**Review by terrajobst — APPROVED** [review]
